Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from datetime import datetime, UTC
+from dateutil.tz import gettz
```

Comment:
use pytz instead - 

def validate_timezone_helper(time_zone):
    """
    Validates Timezone format is correct before assuming it is.
    """
    if time_zone not in pytz.all_timezones:
        return_error(f'Error: Timezone format "{time_zone}" invalid')
    else:
        tz = pytz.timezone(time_zone)
    return tz